getting-started/kubernetes/trying-ebpf.md

```diff
@@ -70,13 +70,7 @@ We recommend using `kubeadm` to bootstrap a suitable cluster on AWS.
 
 1. In AWS, create a controller node and at least 2 worker nodes in the same VPC subnet.  Use Ubuntu 19.10 as the image.
 
-1. **Important** disable the source/destination check on each node's interface.  This is required in order to use non-overlay networking.
-
-   1. Open the Amazon EC2 console and navigate to "Instances".
-
-   1. Select the instance and choose **Actions** > **Networking** > **Change Source/Dest. Check**.
-
-   1. Verify that source/destination check is disabled. Otherwise, choose **Yes, Disable**.
+1. **Important** [Disable Source/Destination Check]({{ site.baseurl }}/reference/resources/felixconfig#spec) on each node's interface.  This is required in order to use non-overlay networking.
```

Comment:
This may be more work than the existing step, so perhaps we should stick with the existing step for this "try out" guide with its very specific number of nodes.